api/api.go

```diff
@@ -0,0 +1,267 @@
+package api
+
+import (
+	"net/http"
+
+	"encoding/json"
+	"fmt"
+	"github.com/ONSdigital/dp-import-api/errors"
+	"github.com/ONSdigital/dp-import-api/models"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/gorilla/mux"
+)
+
+var internalError = "Internal server error"
+
+// ImportAPI - A restful API used to manage importing datasets to be published
+type ImportAPI struct {
+	host      string
+	dataStore DataStore
+	Router    *mux.Router
```

Comment:
this could be an unexported variable (i.e. lowercase router) and then have a GetRouter() method which returns the router. This would avoid accidentally modifying the variable elsewhere